index.js

```diff
@@ -1,55 +1,102 @@
+'use strict';
+
+const PORT = process.env.PORT || 3000;
+
+var Crypto = require('crypto');
+var Fs = require('fs').promises;
+var Http = require('http');
+
+var BodyParser = require('body-parser');
+var Eta = require('eta');
 var express = require('express');
-var eta = require('eta');
-var bodyParser = require('body-parser');
 var morgan = require('morgan');
 
-var app = express();
+var app = require('@root/async-router').Router();
```

Comment:
app.engine doesn't work on this. It does of course on `var app = express();`

Reply:
You need to add it to your package.json